src/overlays/actors/ovl_Bg_Spot09_Obj/z_bg_spot09_obj.c

```diff
@@ -21,25 +24,165 @@ const ActorInit Bg_Spot09_Obj_InitVars = {
     (ActorFunc)BgSpot09Obj_Update,
     (ActorFunc)BgSpot09Obj_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot09_Obj/func_808B1AE0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot09_Obj/func_808B1BA0.s")
+static u32 D_808B1F90[] = { 0x00000000, 0x06005520, 0x0600283C, 0x06008458, 0x06007580 };
+
+typedef s32 (*ActorFuncRet)(struct Actor*, struct GlobalContext*);
+static ActorFuncRet D_808B1FA4[] = {
```

Comment:
Are the casts needed here? I feel like the type should already be correct, and thus I think we could avoid the typedef entirely.
```suggestion
static s32 (*D_808B1FA4[])(BgSpot09Obj* this, GlobalContext* globalCtx) = {
```